### PR TITLE
New v63004/gtm8699 subtest (tests GTM-8699 in V63004)

### PR DESCRIPTION
**Pull request by JakeYottaDB**
$VIEW("STATSHARE",<region>) returns TRUE (1) if the process is currently sharing database statistics for the region and FALSE (0) if it is not.
For the process to be sharing, the database must be enabled for sharing and the process must have opted in to share.
$VIEW("STATSHARE") with no region argument indicates the process has enabled sharing.

### CODE REVIEW

File: v63004/u_inref/gtm8699.csh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+
+echo "# Create a single region DB with region DEFAULT"
+$gtm_tst/com/dbcreate.csh mumps >>& dbcreate_log.txt
+
+echo '# Run gtm8699.m to run $VIEW and V[IEW]'
```
Comment (nars1) [inline v63004/u_inref/gtm8699.csh]: to run -> to test
$VIEW and V[IEW] -> $VIEW("STATSHARE")

File: v63004/inref/gtm8699.m [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+gtm8699()
+
+	WRITE "    $VIEW(""STATSHARE"",""DEFAULT"")",!
+	WRITE "    "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+
+	WRITE "    VIEW ""STATSHARE"" ",!
```
Comment (nars1) [inline v63004/inref/gtm8699.m]: I would like to have a 2 region database file and do VIEW "STATSHARE":"DEFAULT" and verify that $VIEW now shows DEFAULT as sharing whereas it still shows the other region (AREG) as not sharing.

File: v63004/inref/gtm8699.m [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	VIEW "STATSHARE"
+
+	WRITE "    $VIEW(""STATSHARE"",""DEFAULT"")",!
+	WRITE "    "
+	WRITE $VIEW("STATSHARE","DEFAULT"),!
+
+	quit
```
Comment (nars1) [inline v63004/inref/gtm8699.m]: The following note is from the release note. The bolded part needs to be tested too.

For the process to be sharing, **the database must be enabled for sharing** and the process must have opted in to share.

File: v63004/outref/gtm8699.txt [hunk 4]
```diff
@@ -1,7 +1,54 @@
+# Testing for STAT
+
 # Create a single region DB with region DEFAULT
```
Comment (nars1) [inline v63004/outref/gtm8699.txt]: Is "single" a typo? I see multiple regions in play below.

File: v63004/outref/gtm8699.txt [hunk 5]
```diff
@@ -1,7 +1,54 @@
+# Testing for STAT
```
Comment (nars1) [inline v63004/outref/gtm8699.txt]: Testing for STAT -> Testing where database has STATISTICS sharing enabled.

File: v63004/outref/gtm8699.txt [hunk 6]
```diff
@@ -1,7 +1,54 @@
...
+    $VIEW("STATSHARE","AREG"): 1
+    $VIEW("STATSHARE"): 1
+
+    VIEW "NOSTATSHARE" 
+
+    $VIEW("STATSHARE","DEFAULT"): 0
+    $VIEW("STATSHARE","AREG"): 0
+    $VIEW("STATSHARE"): 0
+
+# Testing for NOSTAT
```
Comment (nars1) [inline v63004/outref/gtm8699.txt]: Testing for NOSTAT -> Testing where database has STATISTICS sharing disabled.

File: v63004/u_inref/gtm8699.csh [hunk 7]
```diff
@@ -11,11 +11,50 @@
...
+	$MUPIP set -$share_opt  -reg "*" >>& dbcreate_log.txt
+
+	echo '# Run gtm8699.m'
+	$ydb_dist/mumps -run gtm8699
+
+
+	$gtm_tst/com/dbcheck.csh >>& dbcreate_log.txt
+
+	##### testB ####
+	#echo ''
```
Comment (nars1) [inline v63004/u_inref/gtm8699.csh]: I see various commented out lines. Can they be removed if they don't add any value.

File: v63004/outref/gtm8699.txt [hunk 8]
```diff
@@ -1,6 +1,6 @@
-# Testing for STAT
+# Testing where database has STATISTICS sharing enabled
 
-# Create a single region DB with region DEFAULT
+# Create a 2 region DB with region DEFAULT
```
Comment (nars1) [inline v63004/outref/gtm8699.txt]: Typo : region DEFAULT -> regions DEFAULT and AREG?

**Comment by JakeYottaDB** [comment]
Updates have been made and gtm8699 is ready for review again.

**Pull request merged**